api4/channel_test.go

```diff
@@ -1387,6 +1387,67 @@ func TestConvertChannelToPrivate(t *testing.T) {
 	}
 }
 
+func TestUpdateChannelPrivacy(t *testing.T) {
```

Comment:
Not something needed, but we are pretty big fans of use table testing when is possible and makes sense. Here looks like a table testing can be a good option, but up to you :)

Reply:
Sounds like a great idea. I pushed an update.

Reply:
Nice